ui/jobs/jobs.js

```diff
@@ -2491,6 +2499,105 @@ class Bars {
   }
 
   setupBrd() {
+    // DoT
+    const causticBiteBox = this.addProcBox({
+      id: 'brd-procs-causticbite',
+      fgColor: 'brd-color-causticbite',
+    });
+    const stormbBiteBox = this.addProcBox({
+      id: 'brd-procs-stormbite',
+      fgColor: 'brd-color-stormbite',
+    });
+    [
+      EffectId.Stormbite,
+      EffectId.Windbite,
```

Comment:
Could you leave a comment that iron jaws refreshes these status effects and so doesn't need to be handled separately? I was looking for iron jaws and was a little confused how it worked.

Reply:
You means add this annotation into code right?

Reply:
Yeah, I meant add a comment like `// Iron jaws is automatically handled by refreshing these effects and so we don't need to handle this ability ourselves.`